### DISCUSSION TITLE
I need help regarding the new update of ML4W

### DISCUSSION THREAD

**Discussion by mThanuj** [mylinuxforwork/dotfiles#547] [General]
Is there a way of using older version of ML4W while this new waybar and wallpaper bugs are fixed? please respond fast, i need to get it asap for my works. Thank you in advance

**Comment by mylinuxforwork** [comment]
You can try this https://github.com/mylinuxforwork/dotfiles/issues/546

**Reply by mThanuj** [reply]
hey i know its not related to this. but error: failed to prepare transaction (could not satisfy dependencies)
:: unable to satisfy dependency 'libhyprutils.so=1-64' required by hyprlock. Im getting this error and cant seem to solve the dependency
:(
